util/file_reader_writer.cc

```diff
@@ -738,7 +738,9 @@ class ReadaheadRandomAccessFile : public RandomAccessFile {
     if (s.ok()) {
       buffer_offset_ = offset;
       buffer_.Size(result.size());
-      assert(buffer_.BufferStart() == result.data());
+      if (result.size() > 0) {
+        assert(buffer_.BufferStart() == result.data());
+      }
```

Comment:
I think we need to put `result.size() > 0` into assert.

Reply:
right. done